src/Components/AccountSummaryHome/index.js

```diff
@@ -7,7 +7,7 @@ import PageTitle from '../Common/PageTitle'
 import TranscoderInfo from '../TranscoderInfo'
 import EarnedRewards from '../EarnedRewards'
 import styled from 'styled-components'
-import LPTRewards from '../LPTRewards'
+// import LPTRewards from '../LPTRewards'
```

Comment:
R: Please remove this line (or uncomment it, if needed)

Reply:
Done